.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# trailing whitespace can be significant in markdown
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Maybe .editorconfig should not enforce any style? I have to say that trailing space thing confused me too the first time.